vernac/library.ml

```diff
@@ -640,3 +640,11 @@ let get_used_load_paths () =
        StringSet.empty !libraries_loaded_list)
 
 let _ = Nativelib.get_load_paths := get_used_load_paths
+
+(* Not very safe *)
```

Comment:
This comment is too vague.

Reply:
Yes, In fact I've moved them out of this file to the use point by a second commit.

Reply:
I forgot there is also a `Write State` vernac, I did not move them in the end but improved the comment.